src/test/java/pages/AllSongsPage.java

```diff
@@ -3,36 +3,38 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class AllSongsPage extends BasePage {
 
-    By allSongsLocator = By.cssSelector("a[class='songs']");
-    By songNameLocator = By.xpath("//*[@id=\"songsWrapper\"]/div/div/div[1]/table/tr[1]/td[2]");
-    By playNextButtonLocator = By.xpath("//i[@data-testid='play-next-btn']");
-    By playButtonLocator = By.xpath("//span[@data-testid='play-btn']");
-    By soundBarsLocator = By.cssSelector("img[alt='Sound bars']");
+    @FindBy(css = "a[class='songs']")
+    WebElement allSongsLocator;
+    @FindBy(xpath = "//*[@id=\"songsWrapper\"]/div/div/div[1]/table/tr[1]/td[2]")
+    WebElement songNameLocator;
+    @FindBy(xpath = "//i[@data-testid='play-next-btn']")
+    WebElement playNextButtonLocator;
+    @FindBy(xpath = "//span[@data-testid='play-btn']")
+    WebElement playButtonLocator;
+    @FindBy(css = "img[alt='Sound bars']")
+    WebElement soundBarsLocator;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example


```
   @FindBy(css = "a[class='songs']")
    private WebElement allSongsLocator;
    @FindBy(xpath = "//*[@id=\"songsWrapper\"]/div/div/div[1]/table/tr[1]/td[2]")
    private WebElement songNameLocator;
    @FindBy(xpath = "//i[@data-testid='play-next-btn']")
    private WebElement playNextButtonLocator;
    @FindBy(xpath = "//span[@data-testid='play-btn']")
    private WebElement playButtonLocator;
    @FindBy(css = "img[alt='Sound bars']")
    private WebElement soundBarsLocator;
```